home/ivp-planning/user-interface/conversation-panel.md

```diff
@@ -0,0 +1,9 @@
+# Path Panel
+
+The conversation panel contains all text entries and screenshots and made by users using the [markup tool](../user-interface/markup_tool#conversation-with-Markups).
```

Comment:
The file ending needs to be before the `#` for the anchor to work

Reply:
fixed